packages/functional-tests/pages/settings/index.ts

```diff
@@ -70,7 +75,7 @@ export class SettingsPage extends SettingsLayout {
     const services = await this.connectedServices.services();
     const sync = services.find((s) => s.name.includes(' on '));
 
-    await sync.signout();
+    await sync?.signout();
```

Comment:
Q: What does the `?` do here?
Hhmm, or maybe it isn't a big deal since if `sync` was `undefined` (because `.find()` returned no matches), this would just be `await undefined;` which probably just returns immediately (or maybe on next tick) and is basically a noop.

```js
let sync; // `sync` is currently undefined
await sync?.signout();
// is basically `await undefined`, which doesn't seem to cause issues
// in the browser console.
```

Reply:
Yep, not a big deal - 'sync' was underlined in red since `'sync' is possibly 'undefined'.`